packages/report-server/src/__tests__/reportBuilder/transform/updateColumns.test.ts

```diff
@@ -12,9 +12,9 @@ describe('updateColumns', () => {
       {
         transform: 'updateColumns',
         insert: {
-          number: '1',
-          string: "'Hi'",
-          boolean: 'false',
+          number: '=1',
```

Comment:
This one's kinda a shame huh, I think it might be unexpected that it doesn't automatically interpret numeric only characters as numbers but let's leave it for now and see what users get tripped up on

Reply:
Yeah tis a bit. I suspect we will change this soon enough since sometimes we do things like `insertColumns 'Total': '1'` followed by a `mergeRows` to sum all the totals...

In theory now we could actually move to supporting values other than strings in here as a way to get around that? eg. `{ number: 1 }` instead of `{ number: '1' }`

I chose not to do that right now because:
a) It would probably break a bunch of code in places, and would definitely break the typescript
b) If we're gonna just have a single text input box for this in a UI anyway, we don't want users to need to specify if they're using strings vs raw numbers/booleans, so it might be best to only use strings internally as well

Reply:
Good points, and in general I think we're at a good point in terms of scope of change with this PR, and can work out how we want to handle numbers in the next iteration rather than dragging this out further